test/test_prototype_transforms_functional.py

```diff
@@ -590,6 +590,33 @@ def test_datapoint_explicit_metadata(self, metadata):
             F.clamp_bounding_box(datapoint, **metadata)
 
 
+class TestConvertFormatBoundingBox:
+    @pytest.mark.parametrize(
+        ("inpt", "old_format"),
+        [
+            (next(make_bounding_boxes()), None),
+            (next(make_bounding_boxes()).as_subclass(torch.Tensor), datapoints.BoundingBoxFormat.XYXY),
+        ],
+    )
+    def test_missing_new_format(self, inpt, old_format):
+        with pytest.raises(TypeError, match="new_format"):
+            F.convert_format_bounding_box(inpt, old_format)
+
+    def test_simple_tensor_insufficient_metadata(self):
+        simple_tensor = next(make_bounding_boxes()).as_subclass(torch.Tensor)
+
+        with pytest.raises(ValueError, match="simple tensor"):
```

Comment:
Super nit but it can be helpful for the `match` to indicate what the error is. Perhaps catching "`old_format` has to be passed" would make it clearer. Similarly for the test below, but feel free to not address since we're a bit in  a rush